src/common/utils.ts

```diff
@@ -222,3 +223,35 @@ export const getSchemaSecretsPaths = async (teams: string[]): Promise<string[]>
   cleanSecretPaths.push('users')
   return cleanSecretPaths
 }
+
+async function ensureKeepFile(keepFilePath: string, deps = { writeFile }): Promise<void> {
+  const dirPath = dirname(keepFilePath)
+  if (!(await pathExists(dirPath))) {
+    await $`mkdir -p ${dirname(keepFilePath)}`
+  }
+  if (existsSync(keepFilePath)) return
+  // create the .keep file
+  await deps.writeFile(keepFilePath, '')
+}
+
+export async function ensureTeamGitopsDirectories(envDir: string, deps = { writeFile, glob }) {
+  const dirs = await deps.glob(`${envDir}/env/teams/*`)
+  const gitOpsDirs = ['sealedsecrets', 'workloadValues']
+
+  const keepFilePaths: string[] = []
+  for (const teamDir of dirs) {
+    for (const gitOpsDir of gitOpsDirs) {
+      keepFilePaths.push(join(teamDir, gitOpsDir, '.keep'))
```

Comment:
I think it's more common to add a `.gitkeep` right?
```suggestion
      keepFilePaths.push(join(teamDir, gitOpsDir, '.gitkeep'))
```